packages/fungible-token/bridge-fungible-token/tests/functions/bridge/mod.rs

```diff
@@ -16,6 +16,7 @@ mod success {
 
     use super::*;
 
+    use crate::utils::interface::src20::total_supply;
```

Comment:
I would always merge imports.

What happens if you run `cargo clippy --all-features --all-targets -- -D warnings`?

Reply:
Complains about other things, not this one specifically

```
   Compiling fuel-contract-message-predicate v0.3.0 (/Users/mad/fuel/bridge/packages/message-predicates/contract-message-predicate)
    Checking bridge-fungible-token-contract v0.3.0 (/Users/mad/fuel/bridge/packages/fungible-token/bridge-fungible-token)
error: `assert!(false, ..)` should probably be replaced
   --> packages/fungible-token/bridge-fungible-token/tests/functions/bridge/mod.rs:347:17
    |
347 |                 assert!(false, "Transaction did not succeed")
    |                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |
    = help: use `panic!(..)` or `unreachable!(..)`
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#assertions_on_constants
    = note: `-D clippy::assertions-on-constants` implied by `-D warnings`

error: `assert!(false, ..)` should probably be replaced
    --> packages/fungible-token/bridge-fungible-token/tests/functions/message_receiver/mod.rs:1185:17
     |
1185 |                 assert!(false, "Transaction did not revert");
     |                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
     |
     = help: use `panic!(..)` or `unreachable!(..)`
     = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#assertions_on_constants
```

My go to command with clippy is `cargo clippy --all-features --all-targets --fix`, I 'll make sure to run this one as well.

I 'll push a commit to merge this one, and fix the others that -D warnings gives too

Reply:
Fixed